mindsdb/__main__.py

```diff
@@ -434,18 +434,17 @@ def do_clean_process_marks():
             args=(config.cmd_args.verbose,)
         )
     }
-
-    for api_enum in api_arr:
-        trunc_processes_struct[api_enum].need_to_run = True
-
-    if config['jobs']['disable'] is False:
-        trunc_processes_struct[TrunkProcessEnum.JOBS].need_to_run = True
-
-    if config['tasks']['disable'] is False:
-        trunc_processes_struct[TrunkProcessEnum.TASKS].need_to_run = True
-
     if config.cmd_args.ml_task_queue_consumer is True:
         trunc_processes_struct[TrunkProcessEnum.ML_TASK_QUEUE].need_to_run = True
+    else:
```

Comment:
@StpMax, please take a look. When I start ML_TASK_QUEUE worker it also tries to run http service. I think it is unexpected behavior

Reply:
@ea-rus It is expected. Assumed that the service run on separate machine. In that case http-service responsible for liveness probe.

Reply:
returned back